src/gpu/optixPrograms.cu

```diff
@@ -186,7 +188,6 @@ extern "C" __global__ void __closesthit__()
 
 	// Save sub-sampling results
 	ctx.mrSamples.isHit[mrSamplesIdx] = true;
-	ctx.mrSamples.xyz[mrSamplesIdx] = hitWorldSeenBySensor;
```

Comment:
Just to be sure that this was intended - is this not necessary because we simply sample distance alongside center ray to set xyz of first and last hit?

Reply:
Yes, we don't need to assign here, since we compute XYZ later based on sampled distance.